docs/get-started/v2/get-started/features/noise-cancellation.mdx

```diff
@@ -0,0 +1,90 @@
+100ms Noise cancellation is powered by [krisp.ai](https://krisp.ai/),ensuring clear communication by filtering out background noise.
+
+<Note type='warning'>
+**IMPORTANT** <br/>
+This is an add-on paid feature, for details check [100ms pricing page](https://www.100ms.live/pricing/) 
+</Note>
+
+### Enabling the Noise cancellation
+
+Noise cancellation can be configured from the 100ms dashboard.
+
+#### Enabling the Noise cancellation
+
+1. Navigate to a specific Template where you wish to enable the Noise cancellation.
+2. Click on **‘Advanced Settings’** tab in the Template configuration.
+3. Enable ‘**Noise Cancellation**’.
+
+<video loop="true" autoplay="autoPlay" controls="false" id="vid" muted>
+    <source src="/docs/docs/v2/noise-cancellation/enable-nc.mp4" type="video/mp4" />
+</video>
+
+#### Enabling Noise cancellation by default in preview
+
+For certain roles, you would like to enable Noise cancellation by default from preview state. First enable the Noise cancellation on Template as mentioned above.
+
+1. Navigate to a specific template where you wish to enable the Noise cancellation.
+2. Click on **'Customise Prebuilt'** on top right in the Template.
+3. Click on **'Screens and Components'** and select the role where you wish to enable the noise cancellation by default
+4. Enable **'Noise Cancellation State'**.
+
+<video loop="true" autoplay="autoPlay" controls="false" id="vid" muted>
+    <source src="/docs/docs/v2/noise-cancellation/enable-nc-preview.mp4" type="video/mp4" />
+</video>
+
+### Integrating the Noise Cancellation
+Noise cancellation is available across all platforms (iOS, Android, Flutter, React Native and Web). Refer to the following platform SDK specific guides:
+- [iOS](/ios/v2/how-to-guides/extend-capabilities/plugins/noise-cancellation)
+- [Android](/android/v2/how-to-guides/extend-capabilities/noise-cancellation)
+- [React Native](/react-native/v2/how-to-guides/extend-capabilities/noise-cancellation)
+- [Flutter](/flutter/v2/how-to-guides/extend-capabilities/noise-cancellation)
+- [Web](/javascript/v2/how-to-guides/extend-capabilities/plugins/krisp-noise-cancellation)
+
+### Using Noise cancellation
+Once Noise cancellation is enabled and saved from the template configuration, the Noise cancellation can be used across devices. Noise cancellation can be activated or deactivated in the room or during the preview screen.
+
+<video loop="true" autoplay="autoPlay" controls="false" id="vid" muted>
+    <source src="/docs/docs/v2/noise-cancellation/experience-nc.mp4" type="video/mp4" />
+</video> 
+
+</StepsContainer>
+
+### Frequently Asked Questions (FAQ)
+
+1. **Is Noise cancellation a chargeable feature?**
+    
+    Yes, 100ms Noise cancellation is charged based on per peer usage minutes. This means that for every peer that is enabling noise cancellation during session, their individual usage minutes will be aggregated. For more check [100ms pricing page](https://www.100ms.live/pricing/) or reach out to us.
```

Comment:
For more information, please check [100ms pricing page](https://www.100ms.live/pricing/).

Reply:
Ack